src/org/labkey/test/pages/admin/UpdateShortUrlPage.java

```diff
@@ -0,0 +1,76 @@
+package org.labkey.test.pages.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class UpdateShortUrlPage extends LabKeyPage<UpdateShortUrlPage.ElementCache>
+{
+    public UpdateShortUrlPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static UpdateShortUrlPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("admin", "updateShortURL"));
+        return new UpdateShortUrlPage(webDriverWrapper.getDriver());
+    }
+
+    public String getShortUrl()
+    {
+        return elementCache().shortUrlDisplay.getText().trim();
+    }
+
+    public String setTargetUrl(String targetUrl)
+    {
+        return elementCache().targetUrlInput.get();
+    }
```

Comment:
????
Don't you want to do a set?

Reply:
This page only lets you change the target URL. The shortUrl is read-only.

Reply:
Maybe add a comment before the method stating that?

Reply:
Oh, yes, sorry. I thought this was asking about why there wasn't a `setShortUrl`